internal/messages/composer/input_good_os.go:11:2: cannot find module providing package libdb.so/gotk4-sourceview/pkg/gtksource/v5: import lookup disabled by -mod=vendor
internal/messages/composer/input_good_os.go:12:2: cannot find module providing package libdb.so/gotk4-spelling/pkg/spelling: import lookup disabled by -mod=vendor
vendor/libdb.so/gotk4-spelling/pkg/pkg/spelling/spelling-text-buffer-adapter.go:13:2: cannot find module providing package libdb.so/gotk4-sourceview/pkg/gtksource/v5: import lookup disabled by -mod=vendor

--- internal/messages/composer/input_good_os.go.orig	2024-08-27 09:35:53 UTC
+++ internal/messages/composer/input_good_os.go
@@ -8,8 +8,8 @@ import (
 	"github.com/diamondburned/gotk4/pkg/gtk/v4"
 	"github.com/diamondburned/gotkit/app"
 	"github.com/diamondburned/gotkit/app/prefs"
-	"libdb.so/gotk4-sourceview/pkg/gtksource/v5"
-	"libdb.so/gotk4-spelling/pkg/spelling"
+	"libdb.so/gotk4-sourceview/pkg/pkg/gtksource/v5"
+	"libdb.so/gotk4-spelling/pkg/pkg/spelling"
 )
 
 var spellCheck = prefs.NewBool(true, prefs.PropMeta{
--- vendor/libdb.so/gotk4-spelling/pkg/pkg/spelling/spelling-text-buffer-adapter.go.orig	2024-08-18 08:25:54 UTC
+++ vendor/libdb.so/gotk4-spelling/pkg/pkg/spelling/spelling-text-buffer-adapter.go
@@ -10,7 +10,7 @@ import (
 	coreglib "github.com/diamondburned/gotk4/pkg/core/glib"
 	"github.com/diamondburned/gotk4/pkg/gio/v2"
 	"github.com/diamondburned/gotk4/pkg/gtk/v4"
-	"libdb.so/gotk4-sourceview/pkg/gtksource/v5"
+	"libdb.so/gotk4-sourceview/pkg/pkg/gtksource/v5"
 )
 
 // #include <stdlib.h>
